mk/top_of_makefile.mk

```diff
@@ -41,11 +41,11 @@ else ifeq ($(findstring linux-android,$(TARGET_VENDOR)-$(TARGET_SYS)),linux-andr
 TARGET_ABI = $(TARGET_SYS)
 TARGET_VENDOR = unknown
 TARGET_SYS = linux
-else
+else ifeq ($(findstring redox,$(TARGET_SYS)),redox)
```

Comment:
When double-checking this before merging this, I realized that I don't understand this. What are the target triples for Redox, and in particular isn't the target triple always of the form $(architecture)-$(vendor)-$(os)-$(abi)?, e.g. "i686-unknown-redox-gnu"? If so, what's the point of this change? If not, what are the target triples for redox?